.github/workflows/workflow.yaml

```diff
@@ -248,8 +233,8 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: '1.16.15'
-      - name: 'Run terraform configuration aliases'
```

Comment:
Det kan kanskje være greit å ha med name på alle steps? jeg ble litt forvirra over at noen har name og ikke andre. mulig det er bare meg :sweat_smile:

Reply:
Jeg har tenkt her at actions som ikke trenger videre beskrivelse ikke trenger et navn. Absolutt åpen for andre synspunkter på det. Bra det ikke er breaking :D